.github/workflows/tests-ci.yml

```diff
@@ -113,6 +113,9 @@ jobs:
 
       - name: Install docker
         run: brew install docker && brew install docker-compose # && colima start
+        env:
+          HOMEBREW_NO_AUTO_UPDATE: 1
+          HOMEBREW_NO_INSTALL_CLEANUP: 1
```

Comment:
Curious what the motivation is behind adding this!

Reply:
This is a [well known mac optimization](https://circleci.com/docs/using-macos/#optimizing-homebrew) for CI. Without employing various optimizations like this mac CI envs take between 2x-5x amount of time to run on CI